bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -2113,6 +2118,7 @@ public void safeOperationComplete(int newrc, LedgerMetadata newMeta) {
          */
         private boolean resolveConflict(LedgerMetadata newMeta) {
             LedgerMetadata metadata = getLedgerMetadata();
+            testStubResolveConflict();
```

Comment:
hmm.. i dont get how is this method impacting anyway since you are not using this method return value.

Reply:
this is for the test stub to mock and interject errors at this point. This is LedgerHandle level visibility, hence easy to mock.